schedule/yast/maintenance/create_hdd_transactional_server_restapi.yaml

```diff
@@ -0,0 +1,27 @@
+---
+name: create_hdd_transactional_server
+description: >
+    Installation of a Transactional Server which uses a read-only
+    root filesystem to provide atomic, automatic updates of a
+    system without interfering with the running system.
+vars:
+    DESKTOP: textmode
+    HDDSIZEGB: 20
+    DUD_ADDONS: sdk
+    YUI_REST_API: 1
+schedule:
+    extension_module_selection:
+        - installation/module_registration/register_module_transactional
+    system_role:
+        - installation/system_role/accept_selected_role_transactional_server
+    software: []
```

Comment:
`software` and `security` are not needed once you apply my suggestion in default (only visible ones cannot be the default, that is a special case).

Reply:
actually I got confused myself probably, for default would make more sense to have software: [] so we don't touch this section at all and we override it in the corresponding schedules with the two variations, make sense?  sorry if my previous message confused you...I wasn't seen the whole thing.